### ISSUE TITLE
semantic dice loss usage

### ISSUE THREAD

**Issue by Casperqian** [Joey-S-Liu/MedSAM3#15]
Did you use semantic dice loss during model fine-tuning? I noticed loss_fn_semantic_seg is null in your config file.
I’m wondering if I can directly reuse the semantic segmentation head from your model checkpoint for my task. If not, could you advise on how to adapt the checkpoint or how to use the instance segmentation head for semantic segmentation tasks?

**Comment by Joey-S-Liu** [comment]
> Did you use semantic dice loss during model fine-tuning? I noticed loss_fn_semantic_seg is null in your config file. I’m wondering if I can directly reuse the semantic segmentation head from your model checkpoint for my task. If not, could you advise on how to adapt the checkpoint or how to use the instance segmentation head for semantic segmentation tasks?

Dice loss is used by default in the training script, so there's no need to configure it in the config file.

**Comment by Casperqian** [comment]
It seems that dice loss is only used in the instance segmentation branch in your training config, and the semantic dice loss is not included to optimize the semantic segmentation branch (`loss_fn_semantic_seg: null` in `sam3_lora_configs/lora_base.yaml`). For semantic segmentation, can we directly merge the instance masks into one binary semantic mask?
